FranciscoPerez/react-challenge-project-jan-2023
# Fp 4 fix order user

## Changes
1. Fix the email value passed into the updated auth redux state under the `authReducer` LOGIN action.

## Purpose
Placed orders display `Unknown!` under the `ordered_by` field.

[Issue #4](https://github.com/Shift3/react-challenge-project-jan-2023/issues/4)

## Approach
Within `authReducer.js`, change the email value passed into the updated auth redux state from `action.payload.login` to `action.payload.email`.

This will properly set the `ordered_by` value for placed orders.

## Pre-Testing TODOs
1. **NOTE:**  This branch does not contain fix for Issue #1 - Fix Order Submit

## Testing Steps
1. Navigate to the `/login` page.
2. Enter an email and password.
3. Click `Login` button.
3. Navigate to the `/order` page.
4. Place an order.  (see pre-testing notes)
5. Navigate to the `/view-orders` page.
6. Verify the placed order contains your email under the `Ordered By` field.

## Learning
Applied common troubleshooting steps by tracing through the `loginUser` redux thunk until reaching the `LOGIN` action in `authReducer`.

Closes #4
